test/sslapitest.c

```diff
@@ -4425,6 +4439,11 @@ static int test_key_exchange(int idx)
     if (!TEST_int_eq(SSL_get_shared_group(serverssl, 0),
                      idx == 13 ? 0 : kexch_groups[0]))
         goto end;
+
+    if (!TEST_true(!strcmp(SSL_group_to_name(serverssl, kexch_groups[0]),
```

Comment:
`TEST_str_eq` is probably better here

Reply:
Thanks for the pointer to that function! So now used - again better code.